From ee80095e03efcb6ff62983939c3f7c49c291f12e Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Wed, 19 Sep 2018 20:37:47 +0000
Subject: [PATCH] sysmacros.h: follow changed include files

minor/major is no longer provided by sys/types.h.
explicitely include sysmacros.h where needed.

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 hw/xfree86/common/xf86Xinput.c         | 1 +
 hw/xfree86/os-support/linux/lnx_init.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/hw/xfree86/common/xf86Xinput.c b/hw/xfree86/common/xf86Xinput.c
index 1fb5b16..4fc5a65 100644
--- a/hw/xfree86/common/xf86Xinput.c
+++ b/hw/xfree86/common/xf86Xinput.c
@@ -82,6 +82,7 @@
 #include <stdarg.h>
 #include <stdint.h>             /* for int64_t */
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <sys/stat.h>
 #include <unistd.h>
 #ifdef HAVE_SYS_MKDEV_H
diff --git a/hw/xfree86/os-support/linux/lnx_init.c b/hw/xfree86/os-support/linux/lnx_init.c
index 4acaf33..ceae0f1 100644
--- a/hw/xfree86/os-support/linux/lnx_init.c
+++ b/hw/xfree86/os-support/linux/lnx_init.c
@@ -37,6 +37,7 @@
 #include "xf86_OSlib.h"
 
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 
 #ifndef K_OFF
 #define K_OFF 0x4
-- 
2.13.6

